src-java/northbound-service/northbound-api/src/main/java/org/openkilda/northbound/dto/v2/flows/FlowEndpointV2.java

```diff
@@ -33,5 +34,6 @@
     private Integer portNumber;
     private int vlanId;
     @NonNull
-    private DetectConnectedDevicesV2 detectConnectedDevices;
+    @Default
+    private DetectConnectedDevicesV2 detectConnectedDevices = new DetectConnectedDevicesV2(false, false);
```

Comment:
Can we have no-args constructor in `DetectConnectedDevicesV2`?

Reply:
I think yes. It will be good to have it